web-core-sdk/algorand/react-algorand-core-example/src/App.tsx

```diff
@@ -0,0 +1,192 @@
+import { useEffect, useState } from "react";
+import { Web3AuthCore } from "@web3auth/core";
+import {
+  CHAIN_NAMESPACES,
+  SafeEventEmitterProvider,
+  WALLET_ADAPTERS,
+} from "@web3auth/base";
+import { OpenloginAdapter } from "@web3auth/openlogin-adapter";
+import RPC from "./algorandRPC";
+import "./App.css";
+
+const clientId = "YOUR_CLIENT_ID"; // get from https://dashboard.web3auth.io
+
+function App() {
+  const [web3auth, setWeb3auth] = useState<Web3AuthCore | null>(null);
+  const [provider, setProvider] = useState<SafeEventEmitterProvider | null>(
+    null
+  );
+
+  useEffect(() => {
+    const init = async () => {
+      try {
+        const web3auth = new Web3AuthCore({
+          clientId,
+          chainConfig: {
+            chainNamespace: CHAIN_NAMESPACES.OTHER,
+          },
+        });
+
+        const openloginAdapter = new OpenloginAdapter({
+          adapterSettings: {
+            network: "testnet",
+            uxMode: "popup",
+          },
+        });
+        web3auth.configureAdapter(openloginAdapter);
+        setWeb3auth(web3auth);
+
+        await web3auth.init();
+        if (web3auth.provider) {
+          setProvider(web3auth.provider);
+        }
+      } catch (error) {
+        console.error(error);
+      }
+    };
+
+    init();
+  }, []);
+
+  const login = async () => {
+    if (!web3auth) {
+      console.log("web3auth not initialized yet");
+      return;
+    }
+    const web3authProvider = await web3auth.connectTo(
+      WALLET_ADAPTERS.OPENLOGIN,
+      {
+        loginProvider: "google",
+      }
+    );
+    setProvider(web3authProvider);
+  };
+
+  const getUserInfo = async () => {
+    if (!web3auth) {
+      console.log("web3auth not initialized yet");
+      return;
+    }
+    const user = await web3auth.getUserInfo();
+    console.log(user);
```

Comment:
Can you please change these `console.log` to `uiConsole`?

Reply:
Use it from https://github.com/Web3Auth/examples/blob/main/web-modal-sdk/multi-chain/src/App.tsx#L277

Reply:
Done !